cypress-tests/cypress/e2e/admin/pageBuilder/templates/pageTemplatesCrud.cy.ts

```diff
@@ -0,0 +1,65 @@
+import { customAlphabet } from "nanoid";
+
+context("Page Builder - Blocks", () => {
+    const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz");
```

Comment:
@Asmaloth @adrians5j we have these utils for ID generation neatly packed in our `@webiny/utils` package. We use these extensively in our Jest tests, so please use that, instead of going for raw `nanoid`. Have a look at thow they're implemented: https://github.com/webiny/webiny-js/blob/next/packages/utils/src/generateId.ts

For your needs here, you would just use this:

```
import { generateAlphaId } from "@webiny/utils";

generateAlphaId(6);
```